README.md

```diff
@@ -1,2 +1,30 @@
-# OOP-school-library
-OOP
+# Morse Code
+
+> A OOP school library.
```

Comment:
- [ ][optional]  it would be nice to add a better description for your project, detailing what the project is about.

Reply:
done